frontend/packages/kubevirt-plugin/src/constants/vm/index.ts

```diff
@@ -2,3 +2,8 @@ export * from './constants';
 export * from './network';
 export * from './storage';
 export * from './ui';
+export { PAUSED_VM_MODAL_MESSAGE } from '../../strings/vm/messages';
```

Comment:
ow, no, no, no  no..

Reply:
I don't even know how I let that happen. It's fixed. Now I'm just going to sit over here and hang my head in shame.